Copy icon is been added to copy the content in the dropdown

Hamanth-S opened the pull request:
## Description
Copy icon is been added to copy the content in the dropdown

### Types of changes
Put an x in the boxes that apply. 
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)


### Checklist
Put an x in the boxes that apply.
- [x ] Code compiles correctly
- [ ] All tests passing
- [ ] Extended or added the necessary documentation
- [ ] Added myself / the copyright holder to the AUTHORS file.

Hamanth-S committed:
Copy icon is been added to copy the content in the dropdown

mikeliucc reviewed (changes requested):
for default.css, instead of using `th:nth-child`, we should use more meaning CSS class name instead. This will improve readability and maintenance.

for createTable.js:
1. line 261, remove unnecessary parenthesis
2. line 267-273, possible code simplification with ternary operator
3. line 274, deviate from existing implementation of clipboard (check 'copyToNexial'). Let's make sure we test out thoroughly this use of clipboard. It should not adversely impact or be impacted by existing clipboard functionality.
4.

Hamanth-S committed:
1.No unnecessary paranthesis found
2.Code simplified with Ternary operator
3.The functionality is tested throughly and verfied that there is no impact
